packages/cli/commands/project/dev.js

```diff
@@ -167,19 +166,9 @@ exports.handler = async options => {
     }
   );
 
-  let deployedBuild;
-  let isGithubLinked;
-
-  if (projectExists) {
-    const project = await fetchProject(
-      targetProjectAccountId,
-      projectConfig.name
-    );
-    deployedBuild = project.deployedBuild;
-    isGithubLinked =
-      project.sourceIntegration &&
-      project.sourceIntegration.source === 'GITHUB';
-  }
+  let deployedBuild = project.deployedBuild;
```

Comment:
Is it possible for the project to be undefined here?

Reply:
Ohhh good call, it is